src/pods/employee/employee.vm.ts

```diff
@@ -4,13 +4,19 @@ export interface Employee {
   email: string;
   isActive: boolean;
   temporalPassword?: string;
-  projects: ProjectSummary[];
+  projects: EmployeeProject[];
 }
 
-export interface ProjectSummary {
+export interface EmployeeProject {
   id: string;
   isAssigned?: boolean;
-  projectName: string;
+  name: string;
+}
+
+export interface Project {
```

Comment:
Do you need this interface in VM?

Reply:
From back, i`m feeding a list to project, for that i need this interface.

Reply:
I know, but in VM you are using `EmployeeProject ` not this one

Reply:
Ahhh yes, now i understand, yes i doesn`t need this interface on view model.